SConstruct

```diff
@@ -79,6 +79,7 @@ def run_checks(env):
     if config.CheckHeader('stdatomic.h'):
         env.AppendUnique(CPPDEFINES=['HAVE_STDATOMIC=1'])
 
+    env.AppendUnique(CPPDEFINES=['CRT_PROC_RAW_STRUCT=1'])
```

Comment:
i would rather us not add new ifdefs and split code. aside from initial run for gathering performance data, is there a need to keep this define and associated 'else' code?

Reply:
removed